src/web/js/trove/chart-lib.js

```diff
@@ -771,6 +772,59 @@
     };
   }
 
+  function intervalChart(globalOptions, rawData) {
+    const table = get(rawData, 'tab');
+    const data = new google.visualization.DataTable();
+    data.addColumn('number', 'x');
+    data.addColumn('number', 'values');
+    data.addColumn({id: 'i0', type: 'number', role: 'interval'});
+    data.addColumn({id: 'i1', type: 'number', role: 'interval'});
+
+    const style = get(rawData, 'style');
+
+    const stickColor = get_default_color(rawData);
+    const stickWidth = get(rawData, 'lineWidth');
+
+    const pointColor = get_pointer_color(rawData);
+    const pointSize = get(rawData, 'point-size');
+
+    const fillOpacity = (style == 'boxes') ? 0 : 1;
+
+    data.addRows(table.map(row => [row[0], row[1], row[1], row[2]]));
+
+    const options = {
+      curveType: 'function',
+      lineWidth: 0,
+      intervals: { style:'sticks', lineWidth: 2,  },
+      interval: {
+        'i0': {
+          'style': style,
+          'color': stickColor,
+          'lineWidth': stickWidth,
```

Comment:
Similarly the widths and such, if they could come in as roughnums from Pyret, need to be converted somehow.